### PR TITLE
Move AGL auxiliary layers to meta-xt-images

### PR DESCRIPTION
**Pull request by iartemenko**
Signed-off-by: Iurii Artemenko <iurii_artemenko@epam.com>

### CODE REVIEW

File: inc/agl.inc [hunk 1]
```diff
@@ -15,8 +15,19 @@ XT_QUIRK_UNPACK_SRC_URI += " \
 "
 
 # these layers will be added to bblayers.conf on do_configure
-XT_QUIRK_BB_ADD_LAYER += " \
-    meta-xt-images-domx \
```
Comment (andr2000) [inline inc/agl.inc]: why do you drop meta-xt-images-domx?

Comment (andr2000) [reply inc/agl.inc]: please move meta-xt-images-domx to its original place

Comment (iartemenko) [reply inc/agl.inc]: Done

File: inc/agl.inc [hunk 2]
```diff
@@ -17,6 +17,18 @@ XT_QUIRK_UNPACK_SRC_URI += " \
 # these layers will be added to bblayers.conf on do_configure
 XT_QUIRK_BB_ADD_LAYER += " \
     meta-xt-images-domx \
+    meta-virtualization \
+    meta-selinux \
+    meta-clang \
+"
+
+# Set default revision of AGL auxiliary layers
+BRANCH ?= "master"
```
Comment (andr2000) [inline inc/agl.inc]: I would set it ti morty as it was before

**Review by andr2000 — APPROVED** [review]

File: inc/agl.inc (hunk 2 above)
Comment (iartemenko) [reply inc/agl.inc]: Done

**Review by andr2000 — APPROVED** [review]

**Pull request merged**